algorithms/src/Kokkos_Sort.hpp

```diff
@@ -53,6 +53,11 @@
 
 #include <algorithm>
 
+#if defined(KOKKOS_ENABLE_SYCL)
```

Comment:
This should check if the `oneDPL` TPL was enabled.

Reply:
Do I need it even though oneDPL is `REQUIRED` when `KOKKOS_ENABLE_SYCL` is ON? And if so, should I then add to `FindTPLONEDPL.cmake` something like `add_definitions(-DKOKKOS_ENABLE_ONEDPL)` or is there a better way in Kokkos-cmake-environment?

Reply:
It's not `REQUIRED` if the `oneDPL` can be found natively by the compiler. Add 
```
#cmakedefine KOKKOS_ENABLE_ONEDPL
```
to `cmake/KokkosCore_config.h.in`.

Reply:
Thanks. Done.